features/admin_center_manage_display.feature

```diff
@@ -652,14 +652,36 @@ Feature: Manage Display
     Then I should be able to access 3 "new_secondary_header_link" rows
     And I should not see "Add new secondary header link"
 
-  Scenario: Editing the Visual Design Settings when "Show Redesign Display Settings" and "Use Redesigned Results Page" are true
+  Scenario: Display sub navigation links when "Show Redesign Display Settings" and "Use Redesigned Results Page" are true/false
```

Comment:
@krbhavith The changes here are removing some existing/needed test cases.  Can you take a closer look at this and make sure you aren't removing tests for things like the check for the "preview purposes" blurb and confirmation that the "Visual design (new)" page is reached.